### PR TITLE
Tier 1 Outbounding Review

### PR DESCRIPTION
**Pull request by DinneK**
**Outbound PR**
- All notes are included within the checklist.

### CODE REVIEW

File: CONTRIBUTING.md
Comment (IsaacMilarky) [inline CONTRIBUTING.md]: This list has a typo in it's numbering. All of the items are number 1.

File: README.md
Comment (IsaacMilarky) [inline README.md:40]: These sections shouldn't be blank. It's a Tier 1 so the software delivery lifecycle shouldn't be too detailed. Local Development can link to contributing.md or something.

**Review by IsaacMilarky — CHANGES_REQUESTED** [review]
There are some issues but overall looks good! I'm a bit confused about some things though like: Why are there so many docs in the repodive-tools directory? Why the edits to the cms-gource.conf file? etc.

File: README.md
Comment (sachin-panayil) [reply README.md:40]: added text for both of these

File: CONTRIBUTING.md
Comment (sachin-panayil) [reply CONTRIBUTING.md]: fixed this!

File: code.json
Comment (natalialuzuriaga) [inline code.json]: Just wanted to double check if this is the correct status, are we archiving this?

File: code.json
Comment (natalialuzuriaga) [inline code.json:21]: Do we want to keep this null?

File: code.json
Comment (natalialuzuriaga) [inline code.json:23]: Does this also work in "windows" and "linux"? If so, we should add these values

File: code.json
Comment (natalialuzuriaga) [inline code.json]: Oh nooooo this is formatted incorrectly, I think this is a code.json bug. You can fix manually for now but we should def file a ticket to fix this on repo-scaffolder

File: code.json
Comment (natalialuzuriaga) [inline code.json]: Incorrect formatting, fix manually

File: code.json
Comment (natalialuzuriaga) [inline code.json]: Incorrect formatting, fix manually

File: README.md
Comment (natalialuzuriaga) [inline README.md:41]: 🙌

File: README.md
Comment (natalialuzuriaga) [inline README.md:20]: Link in Core Team section is incorrect, please link to github.com. This came from our old README.md template

File: README.md
Comment (natalialuzuriaga) [inline README.md:48]: It looks like this used an old README.md template so the "Policies" section below is outdated. Please update to: https://github.com/DSACMS/repo-scaffolder/blob/main/tier1/%7B%7Bcookiecutter.project_slug%7D%7D/README.md#policies

File: CONTRIBUTING.md
Comment (natalialuzuriaga) [inline CONTRIBUTING.md:81]: Policy section below is outdated, please update to the following: https://github.com/DSACMS/repo-scaffolder/blob/main/tier1/%7B%7Bcookiecutter.project_slug%7D%7D/CONTRIBUTING.md#policies

File: CONTRIBUTING.md
Comment (natalialuzuriaga) [inline CONTRIBUTING.md]: Let's switch the order of "Building the Project and "Building Dependencies" because first we install/build dependencies then build the project

File: CONTRIBUTING.md
Comment (natalialuzuriaga) [inline CONTRIBUTING.md:61]: I like this text a lot 🙌  We should definitely add some of this content to our templates in repo-scaffolder

File: checklist.md
Comment (natalialuzuriaga) [inline checklist.md]: @IsaacMilarky @decause-gov Should we make the error handling + environment var changes as described here or are we good with moving forward?

**Review by natalialuzuriaga — CHANGES_REQUESTED** [review]
There are some minor updates that need to made to the docs since these markdown files were using old templates. Additionally, definitely double check code.json values with Remy during 1x1 tmr

Otherwise, amazing job you guys with doing your first outbound review, this looks solid 🥳

**Comment by natalialuzuriaga** [comment]
Last couple of notes: 
1. @decause-gov should we add the outbound review checklist to the repo? or delete it?
2. SECURITY.md is outdated too, please update to: https://github.com/DSACMS/repo-scaffolder/blob/main/tier1/%7B%7Bcookiecutter.project_slug%7D%7D/SECURITY.md

File: .github/workflows/checks.yml
Comment (decause-gov) [inline .github/workflows/checks.yml:3]: I suspect we'll continue developing this repo and adding pull requests, so maybe it's a good idea to keep the pull_request checks too? Will defer to the team.

File: CONTRIBUTING.md
Comment (decause-gov) [reply CONTRIBUTING.md]: This is something we'll need to fix in the upstream templates in repo-scaffolder, but +1 Nat.

File: CONTRIBUTING.md
Comment (decause-gov) [reply CONTRIBUTING.md:81]: This will need to be updated in upstream repo-scaffolder then, yeah.

File: README.md
Comment (decause-gov) [reply README.md:48]: plz fix upstream as well 🔧

File: code.json
Comment (decause-gov) [reply code.json]: We're not archiving this one, repodive-tools should be a Tier1, if not a Tier2 project that we will likely update maybe a few times a year?

File: code.json
Comment (decause-gov) [reply code.json:21]: Nah, let's run the new COCOMO estimates here 🥥

File: code.json
Comment (decause-gov) [reply code.json:23]: In theory, it should be able to run on all platforms.

Sachin thinks that this is related to not being able to do 'multi-select' on cookiecutter. Perhaps we can get around this by asking about each platform separately, and then appending them to the field? I don't love that, but it might be a good work around?

**Review by decause-gov — APPROVED** [review]
There's some upstream fixes that need to happen in repo-scaffolder, and some manual updates to a few files here, but after those have been resolved, LGTM +1 🚢

File: checklist.md
Comment (sachin-panayil) [reply checklist.md]: sufficient changes were made over here

File: CONTRIBUTING.md
Comment (sachin-panayil) [reply CONTRIBUTING.md:81]: i believe this is outdated because we were using an old version of the md files that were already in the repo. upstream is updated properly already!

File: README.md
Comment (sachin-panayil) [reply README.md:48]: fixed upstream already!

File: code.json
Comment (sachin-panayil) [reply code.json:23]: wrote this down and will coordinate with Nat for which direction we should take

File: code.json
Comment (sachin-panayil) [reply code.json:21]: ran this again but still getting null. might need to look at something upstream but for now, will leave this as null

File: code.json
Comment (sachin-panayil) [reply code.json]: changed status to production

**Pull request merged**
